_adr/90/index.adoc

```diff
@@ -0,0 +1,136 @@
+---
+num: 90 
+title: Automatic certificate management per Kafka instance 
+status: "Accepted" # One of Draft, Accepted, Rejected
+authors:
+  - "Manyanda Chitimbo"
+tags:
+  - "kafka" # e.g. kafka, connectors, registry
+applies_padrs: # What PADRs does this ADR apply?
+applies_patterns: # What APs does this ADR apply?
+---
+
+// Top style tips:
+// \* Use one sentence per line
+// \* No unexpanded acronyms
+// \* No undefined jargon
+
+// No need for a title heading, it's added by the template
+
+## Context and Problem Statement
+// What is the background against which this decision is being taken?
+The Kafka cluster deployed on the data plane cluster should be reachable via TLS.
+It means that we should provide a way to the clients to verify and validate the TLS connection with the related handshake with the Kafka brokers exposed outside of the data plane cluster.
+The Strimzi operator has its own cluster CA for signing the brokers' certificates but it should be used just for internal communication.
+This problem is the same as the one exposed link:../5/index.adoc[ADR-5: Kafka Cluster Certificate Generation] whose solution only works for managed data plane clusters but it is not an acceptable solution for when users brings / registers their own data plane clusters.
+Additionally, the proposed solution in link:../5/index.adoc[ADR-5] lacked a lot of automation on certificate management which is an area the present ADR intends to address.
+
+## Goals
+// Bulleted list of outcomes that this ADR, if accepted, should help achieve
+- Automated certificate management (generation, renewal, revocation) per Kafka instance
+- Avoid sharing same certificate between Kafka instances
+- Avoid exposing domain registrar & certificate issuers credentials onto enterprise data plane clusters 
+
+## Non-goals
+// Bulleted list of outcomes that this ADR is not trying to achieve.
+- Creating a pattern for and/or developing a central certificate management service
+
+## Current situation
+// Where are we now?
+
+The https://architecture.appservices.tech/adr/5/[ADR-5: Kafka Cluster Certificate Generation] describes the current situation where one certificate is shared across data planes. The certificate is renewed manually using the https://github.com/bf2fc6cc711aee1a0c2a/kas-sre-sops/blob/main/sops/key_management/control_plane/dataplane_certificate.asciidoc[data plane certificate SOP]
```

Comment:
SOP link is to a private repo

Reply:
Yes, we do not have a public / open source friendly SOP - so I guess we'll keep the link as it is for now until we've an equivalent.

Reply:
Ok thanks for the clarification, maybe we can add a note/comment to say that "private link"